lib/util/fs/util.ts

```diff
@@ -3,8 +3,12 @@ import { GlobalConfig } from '../../config/global';
 import { FILE_ACCESS_VIOLATION_ERROR } from '../../constants/error-messages';
 import { logger } from '../../logger';
 
+// http://www.mtu.edu/umc/services/digital/writing/characters-avoid/
+// We allow spaces, but not newlines
+const restricted = /[[\]#%&{}<>*?\b\n\r\0$!'"@|‘“+^`]/;
```

Comment:
```suggestion
const restricted = /[[\]#%&{}<>?\b\n\r\0$!'"@|‘“+^`]/;
```
allow `*` because of wildcards checks?